src/utils/components/Consoles/components/CloudInitCredentials/InlineCodeClipboardCopy.tsx

```diff
@@ -3,21 +3,32 @@ import React, { FC } from 'react';
 import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
 import { ClipboardCopy } from '@patternfly/react-core';
 
-import { clipboardCopyFunc } from '../../utils/utils';
+type InlineCodeClipboardCopyProps = {
+  clipboardText: string;
+  hideText?: boolean;
+};
 
-const InlineCodeClipboardCopy: FC<{ clipboardText: string }> = ({ clipboardText }) => {
+const InlineCodeClipboardCopy: FC<InlineCodeClipboardCopyProps> = ({
+  clipboardText,
+  hideText = false,
+}) => {
   const { t } = useKubevirtTranslation();
 
+  const handleCopy = () => {
+    navigator.clipboard.writeText(clipboardText);
+  };
+
   return (
     <>
       <ClipboardCopy
         clickTip={t('Copied')}
         hoverTip={t('Copy to clipboard')}
         isCode
-        onCopy={clipboardCopyFunc}
```

Comment:
remove the utility function - I think it's not used any more

Reply:
it seems the utility is still there in `src/utils/components/Consoles/utils/utils.ts`